install/linux/usr/share/odemis/sim/sparc2-polarizer-sim.odm.yaml

```diff
@@ -337,6 +337,9 @@ SPARC2: {
     children: {"rz": "Optical Actuators"},
     init: {
         axis_name: "linear",
+    },
+        metadata: {
```

Comment:
Could you indent one level less this line? It's minor, as YAML doesn't care, but it's still nice for reading.